app/src/main/java/org/fossasia/openevent/general/OpenEventDatabase.kt

```diff
@@ -41,7 +43,7 @@ import org.fossasia.openevent.general.ticket.TicketIdConverter
 
 @Database(entities = [Event::class, User::class, SocialLink::class, Ticket::class, Attendee::class,
     EventTopic::class, Order::class, CustomForm::class, Speaker::class, SpeakerWithEvent::class, Sponsor::class,
-    SponsorWithEvent::class, Session::class], version = 6)
+    SponsorWithEvent::class, Session::class, SpeakersCall::class], version = 6)
```

Comment:
I think json file for version = 6 is not committed in development?

Reply:
Yes, it isn't